src/main/resources/static/css/box.css

```diff
@@ -8,4 +8,14 @@
     display: flex;
     justify-content: center;
     margin-top: 90px;
+}
+
+.footer {
```

Comment:
why is this in box.css?

Reply:
should it be in forms.css

Reply:
i think because it is concerning every page and only one css-class so far it's probably best to put it in styles.css